src/main/java/org/jboss/logmanager/BootstrapConfiguration.java

```diff
@@ -0,0 +1,444 @@
+ * {@code org.jboss.logmanager.bootstrap.log.file} system property can be used to override the file the messages will
+ * be written to. If left undefined the default file will be {@code ./bootstrap}.
+ * </p>
+ * <table border="1" cellpadding="4">
+ * <caption style="text-align: left; padding-bottom: 5px"><strong>Available Configuration Properties</strong></caption>
+ * <thead>
+ * <tr>
+ * <th>System Property</th>
+ * <th>Description</th>
+ * <th>Allowed Values</th>
+ * <th>Default Value</th>
+ * </tr>
+ * </thead>
+ * <tbody>
+ * <tr>
+ * <td><code>org.jboss.logmanager.bootstrap.enabled</code></td>
+ * <td>Whether or not bootstrapping should be enabled.</td>
+ * <td><code>true</code> or <code>false</code></td>
+ * <td><code>false</code></td>
+ * </tr>
+ * <tr>
+ * <td><code>org.jboss.logmanager.bootstrap.level</code></td>
+ * <td>The level the root logger should be set to when bootstrapping is enabled. Note that the level used here
+ * will be set on the root logger. If another level is desired on the root logger it must be explicitly set.</td>
+ * <td>Any valid {@linkplain Level level}.</td>
+ * <td><code>INFO</code></td>
+ * </tr>
+ * <tr>
+ * <td><code>org.jboss.logmanager.bootstrap.log.type</code></td>
+ * <td>The type of the handler to use if the JVM exits before {@linkplain LogContext#configurationComplete()
+ * configuration is complete}.</td>
+ * <td><code>console</code> or <code>file</code></td>
+ * <td><code>file</code></td>
+ * </tr>
+ * <tr>
+ * <td><code>org.jboss.logmanager.bootstrap.log.file</code></td>
+ * <td>If the <code>org.jboss.logmanager.bootstrap.log.type</code> is set to <code>file</code> the path to the
+ * log file that should be used.</td>
+ * <td>Any valid path. If the parent directories do not exist they will be created.</td>
+ * <td><code>./bootstrap.log</code></td>
+ * </tr>
+ * <tr>
+ * <td><code>org.jboss.logmanager.bootstrap.queue-size</code></td>
+ * <td>The size of the queue for logged messages until the configuration.</td>
+ * <td>Any valid integer.</td>
+ * <td>10,000</td>
+ * <tr>
+ * <td><code>org.jboss.logmanager.bootstrap.calculate.caller</code></td>
+ * <td>Whether or not the caller should be calculated when the message is queued. Calculating the caller is an
+ * expensive operation. If it is known that the caller will not need to be calculated it is suggested to set
+ * this value to <code>false</code>.</td>
+ * <td><code>true</code> or <code>false</code></td>
+ * <td><code>true</code></td>
+ * </tr>
+ * </tbody>
+ * </table>
+ *
+ * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
+ */
+@SuppressWarnings({"WeakerAccess", "unused"})
+public class BootstrapConfiguration {
+
+    private static final boolean BOOTSTRAP_ENABLED = AccessController.doPrivileged(new PrivilegedAction<Boolean>() {
+        @Override
+        public Boolean run() {
+            return Boolean.parseBoolean(System.getProperty("org.jboss.logmanager.bootstrap.enabled", "false"));
```

Comment:
Maybe just passing `-Dorg.jboss.logmanager.bootstrap.enabled` be passed which would result in an empty string.